app/web/config/tailwind.config.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
       borderRadius: {
         half: '50%',
       },
+      boxShadow: {
+        ch: '0 4px 4px 0 rgba(0, 0, 0, 0.25), 0 4px 4px 0 rgba(13, 13, 19, 0.15)',
```

Comment:
The default box shadows are super low opacity which I think is better suited for lighter themes. I added this for the card.